pkg/scheduler/algorithm/types.go

```diff
@@ -122,6 +123,12 @@ type ReplicaSetLister interface {
 	GetPodReplicaSets(*v1.Pod) ([]*apps.ReplicaSet, error)
 }
 
+// PDBLister interface represents anything that can list PodDisruptionBudget objects.
+type PDBLister interface {
```

Comment:
Is there a particular reason to create a scheduling PDBLister? I think we can directly use `policyv1beta1.PodDisruptionBudgetLister`.

Maybe a technical debt? Because I see other scheduling specific listers (NodeLister/PodLister/ServiceLister/etc.)

Reply:
There is no PodDisruptionBudgetLister in api/policy/v1beta1. The lister is in client-go. I am not sure if creating a dependency on client-go is justified when we only need an interface. Unfortunately, there is one such dependency in "generic_scheduler" which I think we should remove.